### ISSUE TITLE
dragonresearchgroup is ded.

### ISSUE THREAD

**Issue by tapper82** [robzr/sub2rbl#8]
The list for dragonresearchgroup. is ded.

**Comment by Goro2030** [comment]
Working for me today...

**Comment by tapper82** [comment]
root@TappersRouter:~# sub2rbl -l 2 -f stdout
Creating ipset sub2rbl_swing
Retrieving RBL (https://rbldata.interserver.net/ip.txt)
RBL (https://rbldata.interserver.net/ip.txt) added 853 entries
Retrieving RBL (https://lists.blocklist.de/lists/ssh.txt)
RBL (https://lists.blocklist.de/lists/ssh.txt) added 4107 entries
Retrieving RBL (https://lists.blocklist.de/lists/strongips.txt)
RBL (https://lists.blocklist.de/lists/strongips.txt) added 65 entries
Retrieving RBL (https://www.dragonresearchgroup.org/insight/sshpwauth.txt)
Error: retrieving RBL(https://www.dragonresearchgroup.org/insight/sshpwauth.txt): curl: (28) Connection timed out after 15000 milliseconds
Creating ipset sub2rbl-net_swing
Retrieving RBL (https://www.spamhaus.org/drop/drop.txt)
RBL (https://www.spamhaus.org/drop/drop.txt) added 853 entries
Retrieving RBL (https://www.spamhaus.org/drop/edrop.txt)
RBL (https://www.spamhaus.org/drop/edrop.txt) added 130 entries
Destroying ipset sub2rbl_swing
Destroying ipset sub2rbl-net_swing

**Comment by Goro2030** [comment]
You're right... I did a little googling, and looks like the whole Dragon Research group has gone under. I would suggest we take it out of the script.
